src/foam/nanos/cron/Schedulable.js

```diff
@@ -61,13 +110,123 @@
         }
       }
     },
+    {
+      class: 'foam.dao.DAOProperty',
+      name: 'filteredEventDAO',
+      section: 'history',
+      label: 'Scheduled Event History',
+      documentation: 'Show a table view of all historic scheduled events',
+      factory: function() {
+        if ( ! this.eventDaoKey ) return;
+
+        const E = foam.mlang.ExpressionsSingleton.create({});
+        var idPredicate = E.EQ(this.ScriptEvent.OWNER, this.id);
+        var datePredicate = E.LT(this.ScriptEvent.LAST_RUN, new Date());
+
+        return this.__subContext__[this.eventDaoKey].where(datePredicate)
+                                                    .where(idPredicate);
+      },
+      view: function(_, X) {
+        var dao = X.data.filteredEventDAO;
+        return {
+          class: 'foam.comics.v2.DAOBrowserView',
+          config: {
+            class: 'foam.comics.v2.DAOControllerConfig',
+            dao: dao,
+            summaryView: {
+              class: 'foam.u2.table.TableView',
+              columns: [ 'lastRun' ],
+              editColumnsEnabled: false,
+              disableUserSelection: true
+            }
+          }
+        };
+      }
+    },
     {
       class: 'String',
       name: 'objectDAOKey'
     },
     {
       class: 'FObjectProperty',
-      name: 'objectToSchedule'
+      name: 'objectToSchedule',
+      createVisibility: 'HIDDEN'
+    },
+    {
+      class: 'String',
+      name: 'name',
+      section: 'summary',
+      createVisibility: 'HIDDEN',
+      gridColumns: 4,
+      order: 1
+    },
+    {
+      class: 'Date',
+      name: 'nextScheduledDate',
+      section: 'summary',
+      createVisibility: 'HIDDEN',
+      gridColumns: 4,
+      order: 2,
+      javaFactory: `
+        Date d = getSchedule().getNextScheduledTime(getX(), new Date());
+        return d != null ? d : null;
+      `
+    },
+    {
+      class: 'foam.core.Enum',
+      of: 'foam.nanos.auth.LifecycleState',
+      name: 'lifeCycleState',
+      label: 'Status',
+      section: 'summary',
+      createVisibility: 'HIDDEN',
+      gridColumns: 4,
+      order: 3,
+      factory: function() {
+        if (
+          ! this.objectToSchedule ||
+          ! this.objectToSchedule.lifecycleState ||
+          ! this.objectToSchedule.lifecycleState.name
+        ) return;
+        return this.objectToSchedule.lifecycleState.name;
```

Comment:
As above with schedulable being lifecycleAware. Schedulable extends Cron which extends Script, we should be using Script.status.

Reply:
@tharmaman 
I think the design in the ticket describes the status as 'Active' so I was thinking it's a lifecycleAware status.
Because the status in the cron like this below. Please correct me if I am wrong.
<img width="1445" alt="status" src="https://user-images.githubusercontent.com/55885868/180046082-ea1be01f-d9da-4e9a-9450-d2ac31438936.png">